pkgs/tools/package-management/disnix/dydisnix/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, libtool , pkgconfig, glib, libxml2, libxslt, getopt, nixUnstable, disnix }:
+
+stdenv.mkDerivation {
+  name = "dydisnix-0.0";
```

Comment:
0.0 doesn't make much sense. When there's no tagged upstream release, we typically use something like `dydisnix-unstable-2018-04-27` (the date should be the date of the git commit referenced).

Reply:
updated, thx